src/pages/prescreen-1a.tsx

```diff
@@ -55,13 +58,15 @@ export default class Prescreen1c extends React.Component {
 
   handleDateChange(name, value){
     if (name === "birthDatePicked") {
+      console.log('this')
       SessionStore.push("BirthDate", JSON.stringify(value))
       var year62 = new Date(value).getFullYear() + 62;
       SessionStore.push("Year62", year62)
       this.setState({
         birthDate: value
       })
     } else {
+      console.log('that')
```

Comment:
```suggestion
```

Reply:
Whoops, will remove all the console log for debugging.